components/b2b/Lists/CreateList/CreateList.tsx

```diff
@@ -192,17 +192,29 @@ const CreateList = (props: CreateListProps) => {
         <Typography variant="h3" sx={{ fontWeight: 'bold' }}>
           {t('list-items')}
         </Typography>
-        {productList.map((item: CrWishlistItem) => (
-          <ListItem
-            key={item.product?.productCode as string}
-            item={item}
-            onDeleteItem={handleDeleteItem}
-            onChangeQuantity={handleChangeQuantity}
-          />
-        ))}
+        {productList.length === 0 ? (
+          <Typography variant="body2" color="GrayText" marginTop="20px">
+            {t('no-item-in-list-text')}
+          </Typography>
+        ) : (
+          productList.map((item: CrWishlistItem) => (
+            <ListItem
+              key={item.product?.productCode as string}
+              item={item}
+              onDeleteItem={handleDeleteItem}
+              onChangeQuantity={handleChangeQuantity}
+            />
+          ))
+        )}
         {!mdScreen && (
           <>
-            <Box sx={styles.mobileSaveWindow} gap={2}>
+            <Box
+              display={'flex'}
+              flexDirection={'column'}
+              gap={2}
+              marginTop={'20px'}
+              marginBottom={'-70px'}
```

Comment:
don't apply negative margin. Try to make the parent flex and apply gap.

Reply:
I just checked it. The gap at bottom is coming from footer, so removing the negative margin